src/styles/globalStyles.ts

```diff
@@ -37,15 +38,15 @@ const globalStyles = css`
     --h4: calc(var(--h5) / var(--text-scale-ratio)); // 25
     --h5: calc(1em / var(--text-scale-ratio)); // 20
     --small-text: calc(1em * var(--text-scale-ratio)); // 12.8
-    @media screen and (max-width: 1024px) {
+    @media screen and (max-width: 1023px) {
       --text-scale-ratio: 0.85;
     }
-    @media screen and (max-width: 768px) {
+    @media screen and (max-width: 767px) {
```

Comment:
when using max-width, -1px is required

Reply:
for what?

Reply:
Yeah, it should be the other way around. 
max-width for tablet size should be 1024 (or 64r/em), min-width would be +1px

Reply:
min width = "as long as the width is this number or higher, these apply" or, px > min-width
max width = "as long as the width is lower than or equal to this number, these apply, or, px =< max-height

therefore, we need to remove an extra-pixel. But don't take my word for it, use chrome devtools and play around with the tablet sizes

Reply:
I'd agree with Cody here - a difference of one pixel really won't make a difference... So please use a round number :grimacing:

Reply:
what, this is not a matter of opinion lmao, it's the standard viewport of devices, it either works or doesn't, it didn't so I fixed it 😅 😆

Reply:
either that, to switch to min-width, like I do in my own components btw

Reply:
Well aware of what max and min width are, and tablet sizes. If I want to change styles for tablet size and down it would be `max-width: 1024px` because that is the viewport width of the tablet. If the max width is set to 1023 the styles won't be applied

Reply:
so this is the desired bahavior?
```css
.box {
 color: red;
}
@media (max-width: 1024px) {
  .box {
    color: blue;
  }
}
```
so this is the desired result?
- phone: blue
- tablets: blue
- tablets + 1px: red
- large tablets: red
- laptop: blue
- PC: blue

I'd think that the result would be:
- phone: blue
- tablets: red
- tablets + 1px: red
- large tablets: red
- laptop: blue
- PC: blue

Reply:
This whole conversation is a bit pointless now, IMO.

They are just arbitrary numbers... Can we stop bikeshedding now? Thanks :sweat_smile: :100:

If you look at just Apple tablets, then there isn't even a clear consensus... https://yesviz.com/tablets.php So can we please just pick `1024` to appease my round (rem) number needs?